ironfish-rust/src/keys/public_address.rs

```diff
@@ -16,92 +16,76 @@ use std::{convert::TryInto, io};
 
 use super::{IncomingViewKey, SaplingKey};
 
-/// The address to which funds can be sent, stored as a diversifier and public
-/// transmission key. Combining a diversifier with an incoming_viewing_key allows
-/// the creation of multiple public addresses without revealing the viewing key.
-/// This allows the user to have multiple "accounts", or to even have different
-/// payment addresses per transaction.
+/// The generator for public address.
+pub const PUBLIC_KEY_GENERATOR: SubgroupPoint = SubgroupPoint::from_raw_unchecked(
+    bls12_381::Scalar::from_raw([
+        0x3edc_c85f_4d1a_44cd,
+        0x77ff_8c90_a9a0_d8f4,
+        0x0daf_03b5_47e2_022b,
+        0x6dad_65e6_2328_d37a,
+    ]),
+    bls12_381::Scalar::from_raw([
+        0x5095_1f1f_eff0_8278,
+        0xf0b7_03d5_3a3e_dd4e,
+        0xca01_f580_9c00_eee2,
+        0x6996_932c_ece1_f4bb,
+    ]),
+);
```

Comment:
Are we %100 sure this is in the correct prime subgroup + is it known that there aren't any attacks  where people try to move this point into a different coset?